config/config.js

```diff
@@ -192,6 +192,18 @@ var initGlobalConfig = function () {
   // to avoid running test suites on a prod/dev environment (which delete records and make modifications)
   if (process.env.NODE_ENV !== 'test') {
     config = _.merge(config, (fs.existsSync(path.join(process.cwd(), 'config/env/local.js')) && require(path.join(process.cwd(), 'config/env/local.js'))) || {});
+  } else {
+    // Check if we have a local.js custom/local environment. If one exists, we need to check for a test database setting 
+    // that will override the current config object's "db" field. Given the fact that newer versions of MongoDB are 
+    // deprecating the MONGODB-CR authentication. In such cases, mongoose won't connect without the "user" & "pass" options. 
```

Comment:
@Ilanbiala It's not about avoiding an issue, or not.  There are many use cases that this may useful for. 

Think of the reason why we have the `db` setting in *local.js* for use when the app ran with Development. Now apply that same reasoning to someone running the app with Test env.